casper/src/main/scala/coop/rchain/casper/engine/CasperLaunch.scala

```diff
@@ -65,6 +65,7 @@ object CasperLaunch {
       casper <- MultiParentCasper
                  .hashSetCasper[F](init.runtimeManager, validatorId, genesis, init.conf.shardId)
       _ <- Engine.transitionToRunning[F](casper, approvedBlock)
+      _ <- CommUtil.sendForkChoiceTipRequest[F]
```

Comment:
This can work, this I think should be part of `init` method that will be run once `comm` layers gets at least one connection